### PR TITLE
Missing fields

### PR DESCRIPTION
**Pull request by abrande**
- Added Phone and optional title

No tests for Phone currently bc I'm working on signup submission priority

### CODE REVIEW

File: app/components/SignUp/UserInfo.js [hunk 1]
```diff
@@ -38,6 +48,16 @@ const UserInfo = () => {
           required
         />
       </div>
+      <div className="hxCol hxSpan-12">
+        <Field
+          name="phoneNumber"
+          id="phoneNumber"
+          type="number"
```
Comment (explorigin) [inline app/components/SignUp/UserInfo.js]: With normal `<input>` tags there is a [`type="tel"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel) that should be used. `type="number"` will show increment/decrement buttons in some browsers and it's ugly. `type="tel"` is treated like `type="text"` but will bring up a phone pad keyboard on touchscreen devices.

But....since this is not a normal `<input>` tag, I'm not sure how it's being used or if it's even relevant. Edumacate me.

File: webpack.common.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ module.exports = {
     rules: [
       {
         test: /\.(js|jsx)$/,
-        exclude: /node_modules/,
+        exclude: /(node_modules)/,
```
Comment (explorigin) [inline webpack.common.js]: why this change?

File: app/components/SignUp/UserInfo.js (hunk 1 above)
Comment (abrande) [reply app/components/SignUp/UserInfo.js]: Actually that's not used in the component so it's not needed at all here and isn't even being used -- will delete

File: webpack.common.js (hunk 2 above)
Comment (abrande) [reply webpack.common.js]: Ah, I added another exclude, which then was unnecessary so I deleted it. just didn't delete the parenthesis
